solutions/client-landing-zone/setters.yaml

```diff
@@ -125,10 +125,9 @@ data:
   #
   dns-project-id: dns-project-12345
   dns-name: "client-name.example.com."
-  # Select a primary nameserver for the SOA record
-  # The nameservers must be part of the same group of nameservers (a,b,c,d) as the primary nameserver
-  dns-primary-nameserver: |
-    - "ns-cloud-a1.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 259200 300"
+  # The nameservers below must be updated to match the same nameservers as the client DNS subzone
```

Comment:
yeah, that means that this package will have to be deployed twice. once to create the client public dns zone, then you capture the nameservers that have been assigned by google and you update this setter and you deploy again. @lucstjean-ssc can you create a pbi to update the client onboading procedure accordingly.

Reply:
@davelanglois-ssc - Sure, no problem.

Reply:
a good example where I think kpt apply-time-mutation could have been used

Reply:
you can resolve this conversation when you are ready to merge

Reply:
Thanks @davelanglois-ssc

Reply:
@cartyc see above comment related to apply-time-mutation